ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -489,7 +489,7 @@ def setUp(self):
 
         self.limit = datetime.timedelta(seconds=self.test_params['reboot_limit_in_seconds'])
         self.reboot_type = self.test_params['reboot_type']
-        if self.reboot_type not in ['fast-reboot', 'warm-reboot']:
+        if self.reboot_type not in ['fast-reboot', 'warm-reboot', 'warm-reboot -f']:
```

Comment:
Instead of adding another reboot type, I think it is better to detect the upgrade path and add '-f' as needed.

Reply:
Never mind, you are doing it.